.eslintrc.js

```diff
@@ -3,5 +3,18 @@ module.exports = {
     node: true,
     es6: true,
   },
-  extends: ["eslint:recommended"],
+  extends: ["@stellar/eslint-config"],
+  rules: {
+    "import/no-extraneous-dependencies": "off",
```

Comment:
eslintrc config from `https://github.com/stellar/mobile-wallet/blob/master/.eslintrc.js`. There were eslint errors saying  saying `a module should be listed in the project's dependencies.` which seems to be a common error display for react native